### DISCUSSION TITLE
LDM Notes for January 11th, 2023

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#6876] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2023/LDM-2023-01-11.md

## Agenda

- `using` aliases for any types
    - Pointer types in aliases
    - Reference nullability in aliases
    - Value nullability in aliases

**Comment by HaloFour** [comment]
I'm surprised that #90/#1239 wasn't a part of this conversation.

**Reply by 333fred** [reply]
They're out of scope for this proposal. We have liked the idea in the past, but those aren't a (relatively) simple enhancement to an existing feature, they're a major new addition.

**Reply by CyrusNajmabadi** [reply]
Def want to emphasize that we are very interested in generic aliases.  But that feature is many times over more expensive.  Whereas just supporting types in aliases is exceptionally cheap.  So it was a case of just being able to easily get that initial piece out soon

**Reply by HaloFour** [reply]
That's fair, gotta start somewhere.
